docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -0,0 +1,95 @@
+# Release Artifacts
+
+## Status
+
+In Review 06/10/2020
+
+## Context
+
+During the Geneva release of EdgeX Foundry the DevOps WG transformed the CI/CD process with new Jenkins pipeline functionality. After this new functionality was added we also started adding release automation. This new automation is outlined in ADR 0007 Release Automation. However, in ADR 0007 only three release artifact types are outlined. This document is meant to be a living document to try to outline the artifact types for an EdgeX Foundry release and should be modified as that changes.
```

Comment:
**nit**: add link to ADR 0007

Reply:
**suggest**: This document is meant to be a living document which outlines all currently supported artifacts associated with an EdgeX release, and should be updated if/when this list changes.